superset/assets/javascripts/explorev2/components/ChartContainer.jsx

```diff
@@ -186,7 +186,6 @@ class ChartContainer extends React.Component {
           id={this.props.containerId}
           ref={(ref) => { this.chartContainerRef = ref; }}
           className={this.props.viz_type}
-          style={{ overflowX: 'auto' }}
```

Comment:
i'm ok with taking this out. i'll add an issue to investigate how viz's are scaled width wise, to make sure there aren't cases where part  of the viz is cut off.